### DISCUSSION TITLE
Can't find the topic of topic courses

### DISCUSSION THREAD

**Discussion by aldman123** [VikeLabs/courseup#220] [Ideas]
It's rather annoying trying to find what topics courses to take when I need several other tabs open to see what the topic actually is. It would be nice to have it in the UI somewhere.

**Comment by aomi** [comment]
Are you referencing "topic courses" like this?
https://courseup.vikelabs.ca/calendar/202105/SENG?pid=HJkggt6m4
to include:
>TOPICS:SOFTWARE ENGINEER: Quantum Algorithms and Software Engineering 

https://www.uvic.ca/BAN1P/bwckschd.p_disp_detail_sched?term_in=202105&crn_in=32729

**Reply by aldman123** [reply]
Yes exactly

**Comment by keithradford** [comment]
Thanks for bringing this up. Gonna look into this and see what we can do to improve it

**Comment by aomi** [comment]
Made a issue tracking the progress on the scraper.
https://github.com/VikeLabs/uvic-course-scraper/issues/131
